code/modules/mapping/reader.dm

```diff
@@ -31,8 +100,13 @@
 
 	// raw strings used to represent regexes more accurately
 	// '' used to avoid confusing syntax highlighting
-	var/static/regex/dmmRegex = new(@'"([a-zA-Z]+)" = \(((?:.|\n)*?)\)\n(?!\t)|\((\d+),(\d+),(\d+)\) = \{"([a-zA-Z\n]*)"\}', "g")
-	var/static/regex/trimRegex = new(@'^[\s\n]+|[\s\n]+$', "g")
+	var/static/regex/dmmRegex = new(@'"([a-zA-Z]+)" = (?:\(\n|\()((?:.|\n)*?)\)\n(?!\t)|\((\d+),(\d+),(\d+)\) = \{"([a-zA-Z\n]*)"\}', "g")
+	/// Matches key formats in TMG (IE: newline after the \()
+	var/static/regex/matchesTGM = new(@'^"[A-z]*"[\s]*=[\s]*\([\s]*\n', "m")
+	/// Pulls out key value pairs for TGM
+	var/static/regex/varEditsTGM = new(@'^\t([A-z]*) = (.*?);?$')
+	/// Pulls out model paths for DMM
+	var/static/regex/modelPath = new(@'(\/[^\{]*?(?:\{.*?\})?)(?:,|$)', "g")
```

Comment:
while youre here you should make these consistent, like `dmm_regex`, `matches_tgm`, etc.

Reply:
I mean they are consistent but sure